docs/develop/create-and-debug-office-add-ins-in-visual-studio.md

```diff
@@ -195,7 +195,7 @@ Next, Visual Studio does the following:
 
 3. Opens the host application.
 
-Visual Studio doesn't show validation errors in the  **OUTPUT** window when you build the project. Visual Studio reports errors and warnings in the **ERRORLIST** window as they occur. Visual Studio also reports validation errors by showing wavy underlines (known as squiggles) of different colors in the code and text editor. These marks notify you of problems that Visual Studio detected in your code. For more information, see [Code and Text Editor](https://msdn.microsoft.com/library/se2f663y(v=vs.140).aspx). For more information about how to enable or disable validation, see [Options, Text Editor, JavaScript, IntelliSense](/visualstudio/ide/reference/options-text-editor-javascript-intellisense?view=vs-2017).
+Visual Studio doesn't show validation errors in the  **OUTPUT** window when you build the project. Visual Studio reports errors and warnings in the **ERRORLIST** window as they occur. Visual Studio also reports validation errors by showing wavy underlines (known as squiggles) of different colors in the code and text editor. These marks notify you of problems that Visual Studio detected in your code. For more information, see [Code and Text Editor](https://msdn.microsoft.com/library/se2f663y(v=vs.140).aspx). For more information about how to enable or disable validation, see [Options, Text Editor, JavaScript, IntelliSense](/visualstudio/ide/reference/options-text-editor-javascript-intellisense?view=vs-2019).
```

Comment:
Can you confirm if this link (https://msdn.microsoft.com/library/se2f663y(v=vs.140).aspx) is landing the user on a useful/expected page?

Reply:
Kinda makes me wonder if we should scrub docset for MSDN links in general at some point too

Reply:
Yeah, more than likely. Can you add something to the backlog if there isn't already an item?

Reply:
Will do!